src/Dumplie/Application/Command/Customer/AddToCart.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare (strict_types = 1);
+
+namespace Dumplie\Application\Command\Customer;
+
+final class AddToCart
+{
+    /**
+     * @var string
+     */
+    private $sku;
+
+    /**
+     * @var int
+     */
+    private $quantity;
+
+    /**
+     * @var string
+     */
+    private $cartId;
+
+    /**
+     * AddToCart constructor.
+     * @param string $sku
+     * @param int $quantity
+     * @param string $cartId
+     */
+    public function __construct(string $sku, int $quantity, string $cartId)
```

Comment:
I googled what SKU is and a first link tells me: `For example, a tire with a model number of 45790 and a size of 32", may have a SKU of 45790-32.`

So it might not be an integer nor float, but… string? http://www.investopedia.com/terms/s/stock-keeping-unit-sku.asp